src/main/java/matsior/api/beer/BeerServiceImpl.java

```diff
@@ -27,8 +24,8 @@ public List<BeerFullResponse> findAllBeers() {
                 .toList();
     }
 
-    public List<BeerSimpleResponse> findAllBeersSimple(int page, Sort.Direction direction, String sortBy) {
-        return beerRepository.findAll(PageRequest.of(page - 1, PAGE_SIZE, Sort.by(direction, sortBy)))
```

Comment:
Why want to delete this?

Reply:
Tych plików nie ruszałem więc może jakiś alt+enter za duzo usunął nie używaną metodę